dashboard/templates/dashboard/index.html

```diff
@@ -28,9 +28,15 @@
 
                 </ul>
 
-                <a class="nav-link" href='#'>Profile</a>
-                <a class="nav-link" href="{% url 'dashboard:logout' %}">Logout</a>
-
+                <div class="dropdown">
+                    <button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    {{request.session.username}}
```

Comment:
I believe the user story requires for us to have the users' name instead of the username on the dashboard. That way the dashboard is more personalized towards the user.

Reply:
> I believe the user story requires for us to have the users' name instead of the username on the dashboard. That way the dashboard is more personalized towards the user.

I tried using first_name instead, but there is a bug... I cant fix it. It never gives me correct first_name in student/admin object. I think this is related to model.

Reply:
This sounds a bit weird, but having reviewed Pat's PR just now I saw the same reference to the user.first_name and I think that would be working for him. Maybe he'll be able to guide you in a better way than I can for this.

Reply:
I believe it is because sessions is not keeping track of the first name, just the username and that is why only the username can be shown.